### PR TITLE
Fix/amenities

### PR DESCRIPTION
**Pull request by VladimirNee**
repaired display of amenities on accommodationDetails page
added isNew logic to CreateAccommodation

### CODE REVIEW

File: src/axios/axiosInstance.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { SecureStorageKey } from 'src/constants/storage';
 
 import { silentTokenRefresh } from './api';
 import ENDPOINTS from './endpoints';
+import { store } from '../store';
+import { accountActions } from '../store/slices/accountSlice';
 
 const BASE_URL = 'https://dev.vention-booking.taksifon.uz/api';
```
Comment (adadychanka) [inline src/axios/axiosInstance.ts:10]: Can it be stored as an env variable in React Native? 

Can you investigate this question?

Comment (adadychanka) [reply src/axios/axiosInstance.ts:10]: from a quick search 

https://elazizi.com/posts/environment-variables-in-react-native-the-right-way/

File: src/screens/explore/AccommodationDetails/AccommodationDetails.tsx [hunk 2]
```diff
@@ -73,6 +73,41 @@ const AccommodationDetails = ({ route, navigation }: Props) => {
     }
   };
 
+  const amenitiesBadges = useMemo(() => {
+    if (accommodation?.amenities) {
+      const { otherAmenities, id, accommodationId, ...rest } = accommodation?.amenities;
+      const amenities = Object.keys(rest);
+      return amenities.map((amenity) => {
```
Comment (adadychanka) [inline src/screens/explore/AccommodationDetails/AccommodationDetails.tsx]: let's extract this into separate function `function getAmenityBadges(amenities): AmenityChip[]`

File: src/screens/explore/AccommodationDetails/AccommodationDetails.tsx [hunk 3]
```diff
@@ -73,6 +73,41 @@ const AccommodationDetails = ({ route, navigation }: Props) => {
...
+          );
+        }
+      });
+    }
+  }, [accommodation]);
+
+  const otherAmenitiesBadgesMemo = useMemo(() => {
+    if (accommodation?.amenities) {
+      const separator = ', ';
+      if (accommodation?.amenities?.otherAmenities) {
```
Comment (adadychanka) [inline src/screens/explore/AccommodationDetails/AccommodationDetails.tsx]: same here, let's create a function `getExtraAmenityBadges`

File: src/screens/profile/CreateAmenities/CreateAmenities.tsx [hunk 4]
```diff
@@ -86,50 +87,55 @@ const CreateAmenities = ({ route, navigation }: Props) => {
...
-            ...selectedAmenities,
-            otherAmenities: otherAmenities.join(', '),
-          },
-        } as UpdateAmenitiesParams)
-      );
-      if (response.meta.requestStatus === 'fulfilled') {
-        //Placeholder for navigation logic
-      }
-    } catch (error: any) {
+  const onFormSubmitNew = async () => {
```
Comment (adadychanka) [inline src/screens/profile/CreateAmenities/CreateAmenities.tsx]: `New` word can be removed, I suppose

**Review by adadychanka — APPROVED** [review]

File: src/axios/axiosInstance.ts (hunk 1 above)
Comment (VladimirNee) [reply src/axios/axiosInstance.ts:10]: will discuss it with Khushnud and add separately

File: src/screens/profile/CreateAmenities/CreateAmenities.tsx (hunk 4 above)
Comment (VladimirNee) [reply src/screens/profile/CreateAmenities/CreateAmenities.tsx]: removed

File: src/screens/explore/AccommodationDetails/AccommodationDetails.tsx (hunk 2 above)
Comment (VladimirNee) [reply src/screens/explore/AccommodationDetails/AccommodationDetails.tsx]: added functions to the utils

File: src/screens/explore/AccommodationDetails/AccommodationDetails.tsx (hunk 3 above)
Comment (VladimirNee) [reply src/screens/explore/AccommodationDetails/AccommodationDetails.tsx]: added functions to the utils

**Pull request merged**